assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import classNames from 'classnames';
+import { parseISO } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import { DisplayStartOrEndDate } from '@sharedServices/filterState';
+import { PLUS_ONE_MONTH, PLUS_TWO_MONTHS } from '@sharedConstants/defaultDates';
+import { BiggieCalendarDate, CalendarDateRange } from '../';
+
+interface Props {
+	displayDate: DisplayStartOrEndDate;
+	endDate: string;
+	footerText?: string;
+	headerPrefix?: string;
+	headerText?: string;
+	showDate?: boolean;
+	startDate: string;
+	statusClassName?: string;
+}
+
+const CalendarDateSwitcher: React.FC<Props> = React.memo(
+	({ displayDate, footerText, headerPrefix, headerText, statusClassName, ...props }) => {
+		const className = classNames('ee-editor-date-calendar-sidebar', statusClassName);
```

Comment:
I may have made this mistake in the other calendar components in this folder,
but anything in the application folder should not reference a specific use case like "editor-date"
as well, if the idea is for this to be reusable anywhere, then it also should not include "sidebar" in the class name

what if we just pass `statusClassName` through to the child components?
this doesn't add any markup anyways

Reply:
Thx, have cleaned up that.